### PR TITLE
Plot loss instead of accuracy when validating

### PR DESCRIPTION
**Pull request by brett-daley**

### CODE REVIEW

File: net.py [hunk 1]
```diff
@@ -46,13 +46,13 @@ def create_input_pipeline(filenames, batch_size):
...
     image_batch = tf.map_fn(lambda img: tf.image.per_image_standardization(img), image_batch)
 
-    # Flatten the images
-    image_batch = tf.reshape(image_batch, [batch_size, N_PIXELS])
-
     return name_batch, image_batch, label_batch
 
 
 def build_model(image_batch, batch_size):
+    # Flatten the images
```
Comment (brett-daley) [inline net.py]: Moved into build_model() because it's specific to the linear net.

File: main.py [hunk 2]
```diff
@@ -125,7 +130,7 @@ def main():
     parser_train = subparsers.add_parser('train')
     parser_train.add_argument('data_dir', type=str)
     parser_train.add_argument('save_model', type=str)
-    parser_train.add_argument('num_epochs', type=int, default=1)
```
Comment (brett-daley) [inline main.py]: num_epochs shouldn't have a default value. It got reverted when I was fixing merge conflicts.
